### DISCUSSION TITLE
Got an Error in running nuxt project: "'nuxt' is not recognized as an internal or external command"

### DISCUSSION THREAD

**Discussion by Leopard-CC** [community/community#36218] [Actions] [answered]
When I tried to run npm run dev in my nuxt project, my console returned this message:

```
'nuxt' is not recognized as an internal or external command, 
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! andromeda@1.0.0 dev: `nuxt`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the andromeda@1.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Please help me, thanks

**Comment by agiledev31** [comment]
It would be helpful to solve your problem.
I was looking in stackoverflow for similar problems and apparently the problem was the npm cache.

Step 1: npm cache clean --force
Step 2: Delete node_modules
Step 3: npm install

Hope it works for you.

**Reply by Leopard-CC** [reply]
It worked for me
Thank you friend

**Comment by softdream815** [comment]
Make sure nuxt is installed in your Nuxt project:

$ cd /path/to/nuxt-project
$ npm list nuxt
nuxt-project@1.0.0 /path/to/nuxt-project
└── nuxt@2.6.3 
Here /path/to/nuxt-project contains your package.json and node-modules.

If it isn't installed, [add nuxt to your project](https://nuxtjs.org/guide/installation/#installing-code-nuxt-code-) by doing:

$ npm install --save nuxt
Or put it in your project's package.json then do npm install:

  "dependencies": {
    "nuxt": "^2.0.0"
  },
UPDATE:
If you are still getting "nuxt not recognized" problems, try to use explicit path to nuxt from your node_modules directory.

Given this directory (after doing npm install --save nuxt):

nuxt-project
|- node_modules
   |- .bin
      |- nuxt
|- package.json
Update the dev command in package.json with:

"scripts": {
  "dev": "node_modules/.bin/nuxt"
},

**Reply by Leopard-CC** [reply]
Thank you for detailed answer
Looks you're an expert in Nuxt
Would love to learn from you
Let's keep in touch 
Cheers

**Comment by raihanhafiz2015** [comment, marked as answer]
Hi, Everyone!
Hope you're do well.
Please follow this steps:
Step1: Delete node modules
Step2: npm install
Setp3: npm audit fix --force
Thatnks for your attention

**Reply by mullenlv-ashen** [reply]
What a retarded answer
